### PR TITLE
Console K Bar

### PR DESCRIPTION
**Pull request by dvaldivia**
Introduces a K bar for navigating console with keyboard by pressing `cmd + k` on macOS and `ctrl + k` on Linux and Windows.

Also removed a lot of hardcoded colors on the icons.

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by Alevsk — APPROVED** [review]

**Review by adfost — APPROVED** [review]

**Review by bexsoft — CHANGES_REQUESTED** [review]
Found aqn issue with Minio logo
Screen Shot 2022-02-09 at 11 23 18

File: portal-ui/src/icons/CallHomeFeatureIcon.tsx [hunk 1]
```diff
@@ -32,7 +32,6 @@ const CallHomeFeatureIcon = (props: SVGProps<SVGSVGElement>) => (
           data-name="Rectángulo 1614"
           width="6.172"
           height="6.309"
-          fill="#fff"
```
Comment (bexsoft) [inline portal-ui/src/icons/CallHomeFeatureIcon.tsx:35]: Please replace with fill={"currentcolor"} instead of removing the full color

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/icons/CallHomeFeatureIcon.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/icons/CallHomeFeatureIcon.tsx:35]: that's already taken care of in line 24 of this file
https://github.com/minio/console/pull/1532/files/ee2110141860dbb8a046621fb85859a897d6d2cd#diff-b9a01df162b7a19a94894dac6b7e9a9e949e6ec1a2c56b0d7476101b76fd7551L24

**Review by bexsoft — APPROVED** [review]
Talked with @dvaldivia, Changes will be handled in a new PR.

**Comment by harshavardhana** [comment]
🍰

**Pull request merged**